src/workspaces/WorkspacePolicies/WorkspacePolicies.test.tsx

```diff
@@ -0,0 +1,242 @@
+import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { axe } from 'jest-axe';
+import React from 'react';
+import { Link } from 'src/components/common';
+import {
+  azureBillingProject,
+  azureProtectedDataBillingProject,
+  gcpBillingProject,
+} from 'src/testing/billing-project-fixtures';
+import { renderWithAppContexts as render } from 'src/testing/test-utils';
+import {
+  defaultAzureWorkspace,
+  groupConstraintPolicy,
+  protectedAzureWorkspace,
+  protectedDataPolicy,
+  regionConstraintPolicy,
+} from 'src/testing/workspace-fixtures';
+import {
+  AzureWorkspace,
+  groupConstraintLabel,
+  phiTrackingLabel,
+  phiTrackingPolicy,
+  protectedDataLabel,
+  regionConstraintLabel,
+} from 'src/workspaces/utils';
+import { WorkspacePolicies } from 'src/workspaces/WorkspacePolicies/WorkspacePolicies';
+
+describe('WorkspacePolicies', () => {
```

Comment:
TSX conversion resulted in too many changes for this to be recognized as a rename.